Module_Java14/src/main/java/Java14/NewSwitchException.java

```diff
@@ -0,0 +1,30 @@
+package Java14;
+
+public class NewSwitchException {
+    public static void demo() {
+        System.out.println(java14(Month.NOV));
+    }
+
+    public static String java14(Month month) {
```

Comment:
This method is used only in this class. Are you sure that it should has public modificator?